environments/test/test.tfvars

```diff
@@ -1318,5 +1318,20 @@ frontends = [
     custom_domain    = "et-sya.perftest.platform.hmcts.net"
     backend_domain   = ["firewall-nonprodi-palo-cft-perftest.uksouth.cloudapp.azure.com"]
     certificate_name = "wildcard-perftest-platform-hmcts-net"
+  },
+  {
+    product          = "dss-update-case"
+    name             = "dss-update-case"
+    custom_domain    = "dss-update-case.perftest.platform.hmcts.net"
+    backend_domain   = ["firewall-nonprodi-palo-cftperftestappgateway.uksouth.cloudapp.azure.com"]
```

Comment:
This should be `"firewall-nonprodi-palo-cft-perftest.uksouth.cloudapp.azure.com"`

Reply:
thanks @reespozzi , I have done the changes